docker/ci/conan/sanitizer_template.profile

```diff
@@ -10,11 +11,13 @@
 include({{ compiler }})
 
 [options]
-boost/*:extra_b2_flags = "cxxflags=\"{{ sanitizer_build_flags_str }}\" linkflags=\"{{ sanitizer_link_flags_str }}\""
-boost/*:without_stacktrace = True
+boost/*:extra_b2_flags="cxxflags=\"{{ sanitizer_build_flags_str }}\" linkflags=\"{{ sanitizer_link_flags_str }}\""
+boost/*:without_stacktrace=True
 
 [conf]
-tools.build:cflags += {{ sanitizer_build_flags }}
-tools.build:cxxflags += {{ sanitizer_build_flags }}
-tools.build:exelinkflags += {{ sanitizer_link_flags }}
-tools.build:sharedlinkflags += {{ sanitizer_link_flags }}
+tools.build:cflags+={{ sanitizer_build_flags }}
```

Comment:
Out of curiosity, why remove the spaces?

Reply:
It doesn't always apply, but let's be uniform and have it everywhere in the same style.